build.gradle

```diff
@@ -30,6 +31,11 @@ dependencies {
     // logging
     implementation 'ch.qos.logback:logback-classic:1.2.3'
 
+    // db
+    implementation 'org.postgresql:postgresql:42.2.16'
+    implementation 'org.jdbi:jdbi3-core:3.14.4'
+    implementation 'org.flywaydb:flyway-core:7.9.0'
```

Comment:
flyway is the database migration technology that we are going to use.

Reply:
It won't hold back progress, but we should have a discussion on the best way to do DB migrations since the DB schema is relatively stable (actually it's not that stable as I already know of some standardized international address changes I want to make) and is shared across multiple apps (the more important point, I think). Should each app handle its own migrations, or should we just have a shared schema that we manually update when actual DB changes are made? And in that sense, should there be a separate "DB migrations app"?